targets/unity-v2/Testing/Tests/PersistentSockets/PubSubTest.cs

```diff
@@ -28,44 +44,31 @@ public override void ClassTearDown()
         [UUnitTest]
         public void TestPubSubConstruction(UUnitTestContext testContext)
         {
-            //_testContext = testContext;
-            Debug.Log("Starting PubSub UnitTest");
-
-            // specific title id for relay test
-            _previousTitleId = PlayFabSettings.TitleId;
-            PlayFabSettings.TitleId = "70B02F89";
-            PlayFabSettings.VerticalName = "spi-relay";
-
             ClientModels.LoginWithCustomIDRequest login = new ClientModels.LoginWithCustomIDRequest();
-            login.CustomId = "ToddIsTryingToTestPersistentSocketsInUnity" + Guid.NewGuid().ToString();
+            login.CustomId = "PersistentSocketsUnityUnitTest" + Guid.NewGuid().ToString();
             login.CreateAccount = true;
             PlayFabClientAPI.LoginWithCustomID(login, LoginSuccess, LoginFailure, testContext);
         }
 
         void LoginSuccess(ClientModels.LoginResult result)
         {
-            // This needs to be set in PlayFabSettings?... worst case scenario we pass this through the constructor
             _MyEntityKey = new PlayFab.EventsModels.EntityKey { Id = result.EntityToken.Entity.Id, Type = result.EntityToken.Entity.Type };
 
-            // TODO: construct a PubSub object here
             PubSub pubSub = new PubSub(message =>
             {
-                Console.WriteLine(message.ToString());
                 Debug.Log("Received a Message: " + message);
```

Comment:
Now that the test flags itself as passed, the Debug.Log() here is redundant. You know this happened if the test passes.